.github/workflows/cd.yaml

```diff
@@ -95,10 +101,32 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
       - run: |
           set -x
+          export ZQD_AURORA_USER=cimaster
           export ZQD_DATA_URI=s3://zq-ci-master/manual-test
           export ZQD_ECR_HOST=${{ secrets.AWS_ECR_HOST }}
           aws eks --region us-east-2 update-kubeconfig --name zq-test
           kubectl create ns ci-master || :
           kubectl config set-context --current --namespace=ci-master
-          helm uninstall zsrv || :
-          make helm-install
+          make helm-install-with-aurora
```

Comment:
Would still be nice to have this test the postgres kubernetes setup. Perhaps later on, on non-master builds we can do this?

Reply:
Yes, I would be a fan of running these test on all commits. I will figure out a way to do that in a future PR.